pkg/cmd/adm/unregister_member.go

```diff
@@ -62,5 +62,5 @@ func UnregisterMemberCluster(ctx *clicontext.CommandContext, clusterName string)
 	}
 	ctx.Printlnf("\nThe deletion of the Toolchain member cluster from the Host cluster has been triggered")
 
-	return restartHostOperator(ctx, hostClusterClient, hostClusterConfig.OperatorNamespace)
+	return restart(ctx, clusterName)
```

Comment:
the clusterName represents the member that is being unregistered, we want to restart `host` only

Reply:
yes thanks for this out, updated it